src/components/modules/ModuleCheckout.tsx

```diff
@@ -43,29 +43,37 @@ export const ModuleCheckout = (props: ModuleCheckoutProps) => {
 
   const paymentLogoEndMargin: IOSpacingScale = 12;
 
-  return (
-    <PressableModuleBase onPress={props.onPress}>
+  const ModuleBaseContent = (
```

Comment:
Since it is a local component, what do you think about the implementation in this commit (fb1dc45b64cb85ef93554b1baad787035306f647)? Doing in this way, when the `ModuleBaseContent` is invoked, it will be a simple tag `<ModuleBaseContent />`

NB: In the commit fb1dc45b64cb85ef93554b1baad787035306f647 has been fixed also the alignment of the title and CTA button if present a title is very log